headerCheck/factory/coreComponents.go

```diff
@@ -11,13 +13,17 @@ import (
 	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
 )
 
-const marshalizerType = "gogo protobuf"
-const hasherType = "blake2b"
+const (
+	marshalizerType          = "gogo protobuf"
+	hasherType               = "blake2b"
+	hexPubKeyConverterLength = 96
```

Comment:
Can you rename to validatorHexPubKeyLength ?

Reply:
:+1: done